fabric-ca-client/types/index.d.ts

```diff
@@ -2,177 +2,175 @@
-    generateCRL(request: FabricCAServices.IRestriction, registrar: User): Promise<any>;
-    newIdentityService(): FabricCAServices.IdentityService;
-    newAffiliationService(): FabricCAServices.AffiliationService;
-    toString(): string;
+	constructor(url: string | FabricCAServices.IFabricCAService, tlsOptions?: FabricCAServices.TLSOptions, caName?: string, cryptoSuite?: ICryptoSuite);
+	getCaName(): string;
+	register(req: FabricCAServices.IRegisterRequest, registrar: User): Promise<string>;
+	enroll(req: FabricCAServices.IEnrollmentRequest): Promise<FabricCAServices.IEnrollResponse>;
+	reenroll(currentUser: User, attr_reqs: FabricCAServices.IAttributeRequest[]): Promise<FabricCAServices.IEnrollResponse>;
+	revoke(request: FabricCAServices.IRevokeRequest, registrar: User): Promise<any>;
+	generateCRL(request: FabricCAServices.IRestriction, registrar: User): Promise<any>;
+	newIdentityService(): FabricCAServices.IdentityService;
+	newAffiliationService(): FabricCAServices.AffiliationService;
+	toString(): string;
 }
 
-
 export = FabricCAServices;
 
 declare namespace FabricCAServices {
-    export interface TLSOptions {
-        trustedRoots: Buffer | string[];
-        verify: boolean;
-    }
-
-    export interface IFabricCAService {
-        url: string;
-        tlsOptions?: TLSOptions;
-        caName?: string;
-        cryptoSuite?: ICryptoSuite;
-    }
-
-    export interface IKeyValueAttribute {
-        name: string;
-        value: string;
-        ecert?: boolean;
-    }
-
-    export interface IRegisterRequest {
-        enrollmentID: string;
-        enrollmentSecret?: string;
-        role?: string;
-        affiliation: string;
-        maxEnrollments?: number;
-        attrs?: IKeyValueAttribute[];
-    }
-
-    export interface IAttributeRequest {
-        name: string;
-        optional: boolean;
-    }
-
-    export interface IEnrollmentRequest {
-        enrollmentID: string;
-        enrollmentSecret: string;
-        profile?: string;
-        attr_reqs?: IAttributeRequest[];
-        csr?: string;
-    }
-
-    export interface IKey {
-        getSKI(): string;
-
-        /**
+	export interface TLSOptions {
+		trustedRoots: Buffer | string[];
+		verify: boolean;
+	}
+
+	export interface IFabricCAService {
+		url: string;
+		tlsOptions?: TLSOptions;
+		caName?: string;
+		cryptoSuite?: ICryptoSuite;
+	}
+
+	export interface IKeyValueAttribute {
+		name: string;
+		value: string;
+		ecert?: boolean;
+	}
+
+	export interface IRegisterRequest {
+		enrollmentID: string;
+		enrollmentSecret?: string;
+		role?: string;
+		affiliation: string;
+		maxEnrollments?: number;
+		attrs?: IKeyValueAttribute[];
+	}
+
+	export interface IAttributeRequest {
+		name: string;
+		optional: boolean;
+	}
+
+	export interface IEnrollmentRequest {
+		enrollmentID: string;
+		enrollmentSecret: string;
+		profile?: string;
+		attr_reqs?: IAttributeRequest[];
+		csr?: string;
+	}
+
+	export interface IKey {
+		getSKI(): string;
+
+		/**
```

Comment:
Using the tab indent is good for consistency with the rest of the code. The following lines of the comment still look to have spaces as indentation though

Reply:
I lint this ts in another PR #239 
So we could hold this before 239 is merged

Reply:
I apply tsLint to it in another PR #239 
So hold this before 239 is merged